reframe/core/pipeline.py

```diff
@@ -2702,6 +2702,14 @@ class CompileOnlyRegressionTest(RegressionTest, special=True):
     However, if a sanity function is defined, it will be used to validate the
     test.
 
+    .. warning::
+
+       ReFrame checks the exit code of the associated build job to mark the
+       test as a failure. If :attr:`build_locally` is set to :obj:`False` and
+       the scheduler backend cannot retrieve the exit code of the submitted
```

Comment:
Small suggestion, but feel free to ignore. Does it make sense to add `squeue` as an example? Many users may not be aware of this limitation.

Reply:
@ekouts Do you think it's better to move this warning to the documentation of `build_locally`? I think it would have more visibility there, wdyt?

Reply:
We already mention `squeue`'s case there, so it's better if I move it.

Reply:
I agree, it's better in the build_locally documentation.